bin/rspec

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path("../spring", __FILE__)
-rescue LoadError
-end
```

Comment:
full disclosure: you're removing spring here.  Just calling it out. Spring causes so many weird errors.